test/integration/targets/vyos_banner/tasks/cli.yaml

```diff
@@ -8,8 +8,14 @@
 - name: set test_items
   set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
 
-- name: run test case
-  include: "{{ test_case_to_run }}"
+- name: run test case (connection=network_cli)
+  include: "{{ test_case_to_run }} ansible_connection=network_cli"
   with_items: "{{ test_items }}"
   loop_control:
     loop_var: test_case_to_run
+
```

Comment:
Do we need to kill the persistent connection here to avoid it being refused?

Reply:
No, the connection type is taken into account when creating the socket path